hackathons: Add landing page for Athens hackathon (#184)
unikraft/docs: content/en/community/hackathons/2023-03-athens/_index.md

---
title: Athens 2023
date: 2023-01-20T07:20:00+02:00
draft: false
collapsible: true
description: Information on the Athens Unikraft Hackathon, March 30-31, 2023
image: /assets/imgs/2023-03-hack-athens.png
---

## Athens Unikraft Hackathon

Unikraft with [Nubificus](https://nubificus.co.uk/), [the Institute of Communication and Computer Systems (ICCS)](https://www.iccs.gr/en/) and [the National Technical University of Athens (NTUA)](http://www.ntua.gr/index_en.html) come together to organize the **Unikraft Athens Hackathon** to be held on Thursday and Friday, March 30-31, 2023.

> The hackathon will take place as an in-person event at the Multimedia Amphitheatre at the Zografos campus of NTUA.

Reviewer: Actually the area is called Zografou, so it should be "Zografou campus" here and elsewhere. http://old.ntua.gr/map_en.html.

Address is TODO
TODO detailed instructions on reaching the location of the hackathon

Support information and discussions will take place on [Discord](http://bit.ly/UnikraftDiscord) on the `#hack-athens23` channel.

### Registration

To take part in the hackathon please fill this [registration form](https://forms.gle/a315sJrzRQV8rZdz8) by **Sunday, March 19, 2023, 11pm CET**.

> 

Author: Please bring your own laptop. It’s best if you have a native Linux install on your laptop. For any situation, we will have virtual machines ready, to be able to get hacking quickly and easily!

### People

The hosts of the hackathon are:

* [Anastassios Nanos](TODO)
* [Charalampos Mainas](TODO)
* [Michalis Pappas](TODO)
* [Panagiotis Kokkinos](TODO)
* [George Goumas](TODO)
* [Manos Varvarigos](TODO)
* [Nectarios Koziris](TODO)

As part of the Unikraft community [Răzvan Deaconescu](https://github.com/razvand/) (UPB) and [TODO](TODO) will be present onsite with other community members providing support online, on Discord.

### Schedule

#### Thursday, March 30, 2023

| Time (CET)    | Session                                                              | Presenter          |
| ------------- | -------------------------------------------------------------------- | ------------------ |

> | 10:00 - 10:15 | High-level presentation of unikernels and Unikraft                   |                    |

Reviewer 2: ```suggestion
| 10:00 - 10:15 | Overview of unikernels and Unikraft                   |                    |
```

| 10:15 - 11:15 | Introduction to Unikraft. Building and running standard applications |                    |
| 11:15 - 11:30 | Break                                                                |                    |
| 11:30 - 12:45 | Inside Unikraft: Building, configuring, using different libraries    |                    |
| 12:45 - 14:00 | Lunch                                                                |                    |

> | 14:00 - 15:00 | Running (Complex) Application in Unikraft                            |                    |

Reviewer 2: ```suggestion
| 14:00 - 15:00 | Running (complex) applications in Unikraft                            |                    |
```

Author: This is correct. If parentheses were missing, you would write with capital `Complex`.

---
title: Athens 2023
date: 2023-01-20T07:20:00+02:00
draft: false
collapsible: true
description: Information on the Athens Unikraft Hackathon, March 30-31, 2023
image: /assets/imgs/2023-03-hack-athens.jpg
---

## Athens Unikraft Hackathon

> Unikraft, [Nubificus](https://nubificus.co.uk/), [the High Speed Communication Networks Lab (HSCN)](http://hscnl.ece.ntua.gr/) and [the Computing Systems Lab (CSLab)](http://www.cslab.ece.ntua.gr/) from the National Technical University of Athens ([ICCS](https://www.iccs.gr/en/)/[NTUA](http://www.ntua.gr/index_en.html)) come together to organize the **Unikraft Athens Hackathon** to be held on Thursday and Friday, March 30-31, 2023.

Reviewer 3: The [NTUA link](http://www.ntua.gr/index_en.html) gives a 404.

Reviewer: I think https://www.ntua.gr is good enough, as it seems to redirect to https://www.ntua.gr/en if you access from abroad.

Please bring your own laptop.
It's best if you have a native Linux installed on your laptop.
For any situation, we will have virtual machines ready, to be able to get hacking quickly and easily.

### People

The hosts of the hackathon are:

* [Anastassios Nanos](https://ananos.co.uk/)
* [Charalampos Mainas](https://cmainas.gitlab.io/)
* [Michalis Pappas](https://github.com/michpappas)
* [Panagiotis Kokkinos](https://gr.linkedin.com/in/panagiotis-kokkinos-8238783)
* [George Goumas](http://www.cslab.ntua.gr/~goumas/)
* [Manos Varvarigos](https://www.ece.ntua.gr/en/staff/193)
* [Nectarios Koziris](http://www.cslab.ntua.gr/~nkoziris/)

As part of the Unikraft community [Răzvan Deaconescu](https://github.com/razvand/) and [Ștefan Jumărea](https://github.com/StefanJum) will be present on-site, with other community members providing support online, on Discord.

### Schedule

#### Thursday, March 30, 2023

| Time (CET)    | Session                                                              | Presenter          |
| ------------- | -------------------------------------------------------------------- | ------------------ |

> | 10:00 - 10:15 | Overview of unikernels and Unikraft                                  |                    |

Reviewer 3: ```suggestion
| 10:00 - 10:15 | Overview of Unikernels and Unikraft                                  |                    |
```

> | 10:15 - 11:15 | Introduction to Unikraft. Building and running standard applications |                    |

Reviewer 3: ```suggestion
| 10:15 - 11:15 | Introduction to Unikraft: Building and running standard applications |                    |
```

| 11:15 - 11:30 | Break                                                                |                    |
| 11:30 - 12:45 | Unikraft Internals: Building, configuring, using different libraries |                    |
| 12:45 - 14:00 | Lunch                                                                |                    |
| 14:00 - 15:00 | Running (Complex) Application in Unikraft                            |                    |

> | 15:00 - 16:00 | Porting applications to Unikraft                                     |                    |

Reviewer 3: ```suggestion
| 15:00 - 16:00 | Porting Applications to Unikraft                                     |                    |
```

---
title: Athens 2023
date: 2023-01-20T07:20:00+02:00
draft: false
collapsible: true
description: Information on the Athens Unikraft Hackathon, March 30-31, 2023

> image: /assets/imgs/2023-03-hack-athens.png

Reviewer 2: Should the photo be added as well?

---

## Athens Unikraft Hackathon

Unikraft, [Nubificus](https://nubificus.co.uk/), [the High Speed Communication Networks Lab (HSCN)](http://hscnl.ece.ntua.gr/) and [the Computing Systems Lab (CSLab)](http://www.cslab.ece.ntua.gr/) from the National Technical University of Athens ([ICCS](https://www.iccs.gr/en/)/[NTUA](http://www.ntua.gr/index_en.html)) come together to organize the **Unikraft Athens Hackathon** to be held on Thursday and Friday, March 30-31, 2023.

The hackathon will take place as an in-person event at the Multimedia Amphitheater at the Zografou campus of NTUA.
The full address is Multimedia Amphitheater, Central Library Building, Heroon Polytechniou 9, 15780 Zografou, Greece
TODO detailed instructions on reaching the location of the hackathon

Support information and discussions will take place on [Discord](http://bit.ly/UnikraftDiscord) on the `#hack-athens23` channel.

### Registration

> To take part in the hackathon please fill this [registration form](https://forms.gle/a315sJrzRQV8rZdz8) by **Sunday, March 19, 2023, 11pm CET**.

Reviewer 2: The photo in the form is wrong. It shows details about the Munich Hackathon.

> To take part in the hackathon please fill this [registration form](https://forms.gle/a315sJrzRQV8rZdz8) by **Sunday, March 19, 2023, 11pm CET**.

Author: fixed
